docs/architecture-decisions/cucumber-gherkin-test-suite.md

```diff
@@ -0,0 +1,78 @@
+---
+status: accepted
```

Comment:
Please note I'm starting this as "accepted" because this is a "retrospective" ADR, if you disagree, let me know.